### PR TITLE
addRef

### PR DESCRIPTION
**Pull request by peterstuart**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -46,15 +46,20 @@ const App: FC<AppProps> = flow(
     ['name'],
   ),
   addProps(({ name }) => ({ doubledName: `${name} ${name}` }), ['name']),
+  addRef('containerRef', null as HTMLDivElement | null),
```
Comment (peterstuart) [inline src/App.tsx]: You have to be specific here, which I guess is fine?

Comment (helixbass) [reply src/App.tsx]: Ya that seems expected

File: types/ad-hok/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 declare module 'ad-hok' {
+  import { MutableRefObject } from 'react'
```
Comment (helixbass) [inline types/ad-hok/index.d.ts]: So is this where you were saying now you understand importing types from external libraries better?

Comment (peterstuart) [reply types/ad-hok/index.d.ts]: Yeah, I initially just had my own compatible interface, but this is better.

**Pull request merged**